pkg/ingester/ingester_v2.go

```diff
@@ -607,6 +607,7 @@ func (i *Ingester) createTSDB(userID string) (*userTSDB, error) {
 		RetentionDuration: uint64(i.cfg.TSDBConfig.Retention / time.Millisecond),
 		BlockRanges:       i.cfg.TSDBConfig.BlockRanges.ToMilliseconds(),
 		NoLockfile:        true,
+		StripeSize:        int(1 << uint(i.cfg.TSDBConfig.StripeSize)),
```

Comment:
In addition to validation that Marco suggested, should we also check for `<=0` and either error out or use the default `1 << 14` in that case?

Reply:
Done.